recipes/pysis/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pysis" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/USGS-Astrogeology/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: ec1561563750293daa3d6d0afdd80989de3064b1d21d2a0a86c092b4c86aeeb3
+
+build:
+  skip: True  # [win]
+  number: 0
+  script:
+    - bash tests/make_isis_env.sh
+    - python setup.py install
+
+requirements:
+  build:
+    - python
+    - numpy
+    - six
+    - pvl
+    - pytest
```

Comment:
Do you really need ``pytest`` to build this package? or just to test this library?

Reply:
Just for testing, this release has already been tested using CI so this line can just go.

Reply:
Please remove ``pytest`` then